PersonApi/V1/Factories/EntityFactory.cs

```diff
@@ -45,11 +46,16 @@ public static PersonDbEntity ToDatabase(this Person entity)
                 PlaceOfBirth = entity.PlaceOfBirth,
                 DateOfBirth = entity.DateOfBirth,
                 Gender = entity.Gender,
-                Identifications = entity.Identifications,
-                Languages = entity.Languages,
-                CommunicationRequirements = entity.CommunicationRequirements,
-                PersonTypes = entity.PersonTypes
+                Identifications = entity.Identifications.ToList(),
+                Languages = entity.Languages.ToList(),
+                CommunicationRequirements = entity.CommunicationRequirements.ToList(),
+                PersonTypes = entity.PersonTypes.ToList()
             };
         }
+
+        public static string NormaliseDbId(Guid id)
```

Comment:
Should this method be here, and is there a reason for it to be public?

Reply:
I don't mind it there :) Just making sure it wasn't let for no reason (i miss stuff like that in my own code sometimes.. )

Reply:
It's now gone anyway. :-)